pavelib/quality.py

```diff
@@ -161,20 +161,18 @@ def _get_pylint_violations(systems=ALL_SYSTEMS.split(','), errors_only=False, cl
 @cmdopts([
     ("system=", "s", "System to act on"),
     ("errors", "e", "Check for errors only"),
-    ("limit=", "l", "Limits for number of acceptable violations - either <upper> or <lower>:<upper>"),
 ])
 @timed
 def run_pylint(options):
     """
     Run pylint on system code. When violations limit is passed in,
     fail the task if too many violations are found.
     """
-    lower_violations_limit, upper_violations_limit, errors, systems = _parse_pylint_options(options)
     errors = getattr(options, 'errors', False)
     systems = getattr(options, 'system', ALL_SYSTEMS).split(',')
     result_name = 'pylint_{}'.format('_'.join(systems))
 
-    num_violations, _ = _get_pylint_violations(systems, errors)
+    num_violations, violations_list = _get_pylint_violations(systems, errors)
```

Comment:
Getting the `violations_list` to show the warnings in the console output in case of failure.